civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,195 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
```

Comment:
Is `.contact-id` custom or auto generated?

Reply:
If yes then we need to use BEM
`.chr_panel--my-details__personal__contact_id`
 
Similarly name 
```
.contact-id,
.gender,
.marital-status
```